pubspec.yaml

```diff
@@ -17,10 +17,10 @@ dependencies:
 dev_dependencies:
   analyzer: ^7.4.5
   coverage: ^1.13.1 # See: https://github.com/dart-lang/test/issues/1265
-  ffigen: ^18.1.0
+  ffigen: ^19.0.0
```

Comment:
https://pub.dev/packages/ffigen/changelog#1900

> Breaking change: Major change to the way ObjC methods are code-genned. Methods now use named parameters, making them more readable and closer to how they're written in ObjC. For example, the NSData method dataWithBytes:length: used to be generated as dataWithBytes_length_(Pointer<Void> bytes, int length), but is now generated as dataWithBytes(Pointer<Void> bytes, {required int length}). Protocol methods are not affected.
Migration tip: A quick way to find affected methods is to search for _(.